RELEASE/scripts/autoscend/auto_pre_adv.ash

```diff
@@ -325,11 +325,113 @@ void handlePreAdventure(location place)
 		}
 	}
 
+
+	// ML adjustment zone section
+	boolean doML = true;
+	boolean removeML = false;
+		// removeML MUST be true for purgeML to be used. This is only used for -ML locations like Smut Orc, and you must have 5+ SGEAs to use.
+		boolean purgeML = false;
+
+	boolean[location] highMLZones = $locations[Oil Peak, The Typical Tavern Cellar, The Haunted Boiler Room];
+	boolean[location] lowMLZones = $locations[The Smut Orc Logging Camp];
+
+	// Generic Conditions
+	if(get_property("kingLiberated").to_boolean())
+	{
+		doML = false;
+	}
+
+	if(((get_property("flyeredML").to_int() > 9999) || get_property("auto_hippyInstead").to_boolean() || (get_property("auto_war") == "finished") || (get_property("sidequestArenaCompleted") != "none")) && ((my_level() >= 13)))
```

Comment:
```
my_level() >= 13
```
Wondering how people feel about the idea of only setting doML = false if `my_level() == 13` instead of `my_level() >= 13`, I ask because often in aftercore i want to be level 15 to do dread or be as strong as possible to make the sea easier (cheaper). I believe you only get extra karma if you kill NS at level 13, so if we miss that, might as well gain some extra stats right? Risk of getting killed but by that point we can generally take anything pretty easily. Its also not that hard to hit 15 in aftercore with scaling monsters, so would just potentially save adventures/meat in aftercore or make NS fight easier.

Reply:
Good point, no use locking the barn door after the horse escaped. Done and added a note.